Firestore/core/src/firebase/firestore/util/exception.h

```diff
@@ -69,6 +87,17 @@ using ThrowHandler = void (*)(ExceptionType type,
                               int line,
                               const std::string& message);
 
+/**
+ * The default throw handler implementation, suitable for C++. If exceptions are
+ * enabled, throws a C++ exception, except for AssertionFailureNoThrow, which
+ * unconditionally aborts.
```

Comment:
Ultranit: s/aborts/terminates/? Not that it's a very important distinction in our case, but the current wording suggests we call `std::abort`.